.travis.yml

```diff
@@ -1,16 +1,44 @@
 sudo: required
 dist: trusty
 
+# Notes:
+#   * 2019-02: trusty: distro Octave is 3.8
+#   * 2019-02: xenial: snapshots don't work, distro Octave is 4.0
```

Comment:
Yeah, sad but true. At some point I think we should migrate doctest and octsympy CI to use docker, gives full control over the OS and the packages installed, and I think Travis caches docker layers.